ldk/javascript/typedoc.json

```diff
@@ -1,18 +1,12 @@
 {
   "out": "docs",
-  "includeDeclarations": true,
   "excludeExternals": true,
   "excludePrivate": true,
-  "mode": "file",
-  "toc": [],
   "exclude": ["coverage/**/*", "jest.config.js", "**/*.test.ts"],
-  "listInvalidSymbolLinks": true,
-  "inputFiles": ["src/index.ts"],
-  "stripInternal": true,
-  "theme": "pages-plugin",
-  "name": "Javascript Loop Development Kit",
-  "pages": {
-    "groups": [],
-    "theme": "pages-plugin"
-  }
+  "validation": {
+    "invalidLink": true
+  },
+  "entryPoints": ["src/index.ts"],
+  "entryPointStrategy": "expand",
+  "name": "Javascript Loop Development Kit"
```

Comment:
This is annoying request (especially since you didn't create this text), but can you change `Javascript` -> `JavaScript`?

Reply:
yeah definitely